.github/workflows/pr-labelling.yaml

```
@@ -0,0 +1,12 @@
+name: Continuous Integration
+run-name: CI - Test and Coverage Check by @${{ github.actor }}
+on:
+  pull_request:
+    types: [ opened, synchronize, reopened ]
+jobs:
+  apiDocumentation:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Label PR based on file changes
+        uses: actions/labeler@v5
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)